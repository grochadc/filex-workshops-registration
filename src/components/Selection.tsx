import React from "react";

type Student = {
  name: string;
};

type SelectionProps = { student: Student };

const Selection: React.FC<any> = ({ student }: SelectionProps) => {
  return <div>Hello {student.name}!</div>;
};

export default Selection;
