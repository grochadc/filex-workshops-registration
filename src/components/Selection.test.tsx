import React, { Fragment } from "react";
import { render } from "@testing-library/react";
import Selection from "./Selection";

test("Renders without crashing", () => {
  const { asFragment } = render(<Selection student={{ name: "Pedro" }} />);
  expect(asFragment()).toMatchSnapshot();
});
