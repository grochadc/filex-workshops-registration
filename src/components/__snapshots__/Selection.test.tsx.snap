// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without crashing 1`] = `
<DocumentFragment>
  <div>
    Hello Pedro!
  </div>
</DocumentFragment>
`;
